Execution failed for task ':react-native-fused-location:verifyReleaseResources'.

when i'm doing "./gradlew assembleRelease"

this error occurs:

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-fused-location:verifyReleaseResources'.
> com.android.ide.common.process.ProcessException: Failed to execute aapt

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -10,13 +10,17 @@ buildscript {
 
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.1"
+    compileSdkVersion safeExtGet('compileSdkVersion', 25)
+    buildToolsVersion safeExtGet('buildToolsVersion', '25.0.1')
 
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 25
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 25)
         versionCode 1
         versionName "0.1"
     }

